group-work.md

```diff
@@ -16,11 +16,56 @@ questions:
 
 ---
 
-## Step X: Contributing non-trivial changes
+## GitHub organizations
 
-Submit a larger change using GitHub Desktop.  As an example our task will be to
-add a new file (in our case a taco recipe) to the exercise repository.
+Should I start a repository under my account or open a new organization?
+- Single-person projects often start under own account.
+- If is no problem to move a project from own namespace to an organization later.
+- When starting a larger project with several people, possibly several affiliations, an organization may be a better start.
+- If this is a GitHub pages project, then it will matter for the URL:
+  `myuser.github.io/myproject/` vs. `ourorg.github.io/ourproject/`.
 
+Should I add everybody as collaborator?
+- If you are a handful of project collaborators it probably makes sense to add everybody as collaborators.
+- But one does not have to be a *collaborator* to contribute (anybody can send contributions to public projects).
+- External contributors don't have to be added.
+
+---
+
+## Organizational permissions
+
+- Organizations have **owners** and **members**.
+- Owners can add more members and delete repositories.
+- Members can also be organized into teams.
+- We recommend to write-protect the default branch and protect it against force-pushes and accidental deletions.
+
+---
+
+## GitHub or GitLab?
+
+- GitHub: probably better integrations (with services like Zenodo), probably more visibility (more users).
+- GitLab: more features, you can also self-host, more advanced continuous integration.
+
+---
+
+## Direct commits or pull requests?
+
+- For single-person projects: direct pushes.
+- If you have somebody who can help you with code review: use pull requests.
+- For projects with 2 or more persons: agree on applying all changes via pull requests
+  and create a new branch for every change.
+
+---
+
+## Small vs. large changes
+
+- For larger changes first open an issue and describe your idea and collect feedback.
+- Alternative: if you already have a larger change half-finished but you are unsure whether you are on the right
+  track, open a **draft pull request**. These are meant to share unfinished drafts and collect suggestions.
```

Comment:
Not necessarily to write down, but if I would present I would again ask "do you ever review what others do?  Would it improve your work if you did?  Would it overall save time?"

Reply:
I will add it there as memory help.